custom_dc/README.md

```diff
@@ -32,6 +32,58 @@ Additionally, a custom DC combines its own local datasets with base DC datasets
   You can use a sample key `AIzaSyA9RsPS8tBCKwrET3qqAzydhOMWP0Ee8Y8` for local
   development as discussed below.
 
+## Quick Start
+
+Once the prerequisites and api keys from above are in place, 
+here's how you can start a local custom DC instance quickly.
+
+### Env variables
+
+Open [sqlite_env.list](sqlite_env.list) and specify values for `DC_API_KEY` and `MAPS_API_KEY`.
+
+> NOTE: Do not use double-quotes or spaces when specifying the values.
+
+### Run docker
```

Comment:
Instead of "Run docker", maybe "Start service"?

Reply:
Done.